analyses/metacells/README.md

```diff
@@ -9,20 +9,38 @@ Initially, this module will contain some scripts and notebooks for applying anal
 
 ## Usage
 
-TBD
+To run the module, first install and activate the `openscpca-metacells` conda environment.
+From the module directory, run:
+
+```sh
+conda-lock install --name openscpca-metacells
+conda activate openscpca-metacells
+```
+
+Then run the main bash script:
+
+```sh
+bash run-metacells.sh
+```
+
 
 ## Input files
 
 Initial analysis is expected use processed `SingleCellExperiment` objects from the ScPCA Portal, in AnnData format.
 
+For initial testing, files were downloaded with:
+
+```sh
+./download-data.py --format "anndata" --projects SCPCP000001
+```
 
 ## Output files
 
-TBD
+Currently, the script outputs three files per library: an updated AnnData object, a pickled SEACells model object, and a log file.
```

Comment:
Currently it looks like it only outputs a logfile if one specified

Reply:
Which it is in the run script!

Reply:
yeah, fair! Since I was mostly reviewing python, "the script" meant the python script to me, but that's not what it means here! 👍